rotkehlchen/tests/unit/test_calendar_reminders.py

```diff
@@ -141,13 +142,27 @@ def test_ens_expiry_calendar_reminders(
     calendar_db = DBCalendar(database)
     all_calendar_entries = calendar_db.query_calendar_entry(CalendarFilterQuery.make())
     assert all_calendar_entries['entries_total'] == 0
+    original_requests_get = requests.get
 
-    ens_events = [
-        next(x for x in get_decoded_events_of_transaction(  # decode ENS registration/renewal event and get the event with the metadata  # noqa: E501
-            evm_inquirer=ethereum_inquirer if counterparty == CPT_ENS else base_inquirer,
-            tx_hash=ens_tx_hash,
-        )[0] if x.extra_data is not None) for ens_tx_hash in ens_tx_hashes
-    ]
+    def mock_basenames_request_get(url, timeout):
+        """There is a basenames api request that fails and is excluded from the VCR. So force
```

Comment:
hmm this does not make sense. This test is not VCRed. I don't understand the comment.